Formula/leafnode.rb

```diff
@@ -14,6 +14,89 @@ def install
                           "--with-user=#{ENV["USER"]}", "--with-group=admin",
                           "--sysconfdir=#{etc}/leafnode", "--with-spooldir=#{var}/spool/news/leafnode"
     system "make", "install"
+    (prefix/"homebrew.mxcl.fetchnews.plist").write fetchnews_plist
+    (prefix/"homebrew.mxcl.texpire.plist").write texpire_plist
+  end
+
+  def caveats; <<-EOS.undent
+    For starting fetchnews and texpire, create links,
+      ln -s #{opt_prefix}/homebrew.mxcl.{fetchnews,texpire}.plist ~/Library/LaunchAgents
+    And to start the services,
+      launchctl load -w ~/Library/LaunchAgent/homebrew.mxcl.{fetchnews,texpire}.plist
+    EOS
+  end
+
+  plist_options :manual => "leafnode"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>OnDemand</key>
+        <true/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>Program</key>
+        <string>#{opt_sbin}/#{name}</string>
```

Comment:
Just use `leafnode` rather than `#{name}`. I'm never convinced `#{name}` usage is wildly obvious to people new to Ruby.